examples/api/lib/material/refresh_indicator/refresh_indicator.2.dart

```diff
@@ -0,0 +1,92 @@
+import 'package:flutter/material.dart';
+
+/// Flutter code sample for [RefreshIndicator.noSpinner].
+
+void main() => runApp(const RefreshIndicatorExampleApp());
+
+class RefreshIndicatorExampleApp extends StatelessWidget {
+  const RefreshIndicatorExampleApp({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return const MaterialApp(
+      home: RefreshIndicatorExample(),
+    );
+  }
+}
+
+class RefreshIndicatorExample extends StatelessWidget {
+  const RefreshIndicatorExample({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: const Text('RefreshIndicator.noSpinner Sample'),
+      ),
+      body: RefreshIndicator.noSpinner(
+        // Callback function used by the app to listen to the
+        // status of the RefreshIndicator pull-down action.
+        onStatusChange: (RefreshIndicatorStatus? status) {
+          print('RefreshIndicatorStatus was updated to: $status');
+
+          // Do something when the status changes
+        },
+        onRefresh: () async {
+          print('RefreshIndicator was called!');
+
+          // Replace this delay with the code to be executed during refresh
+          // and return asynchronous code
+          return Future<void>.delayed(const Duration(seconds: 3));
+        },
+
+        // This check is used to customize listening to scroll notifications
```

Comment:
This check? 

Better to explain this a little more directly: To ensure that we're notified only when the nested scroll view is scrolled we check the notification's depth. [Now explain what the default is and why we don't want that, etc.]

Reply:
I copied part of the sample from another refresh indicator example, maybe I'll just remove this part?